contrib/epee/include/storages/http_abstract_invoke.h

```diff
@@ -32,6 +32,9 @@
 #include "portable_storage_template_helper.h"
 #include "net/http_base.h"
 #include "net/http_server_handlers_map2.h"
+#if (defined(__ANDROID__))
+#include <android/log.h>
+#endif
```

Comment:
Removed that commit. Not really needed yet. Tried to patch easylogger++ to make it print to android debug bridge instead, but without success so far.